src/mono/mono/mini/mini-llvm.c

```diff
@@ -306,7 +306,7 @@ static LLVMIntPredicate cond_to_llvm_cond [] = {
 };
 
 static LLVMRealPredicate fpcond_to_llvm_cond [] = {
-	LLVMRealOEQ,
+	LLVMRealUEQ,
```

Comment:
Do all these cases need to be changed ?

Reply:
Only one place should be changed. I reverted the unwanted change.